packages/rn-tester/README.md

```diff
@@ -29,10 +36,14 @@ yarn clean-ios
 If you are still having a problem after doing the clean up (which can happen if you have built RNTester with older React Native versions where files were generated inside the react-native folder.), the best way might be to clean-install react-native (e.g. remove node_modules and yarn install).
 
 Both macOS and Xcode are required.
-1. `cd packages/rn-tester`
-2. Install [Bundler](https://bundler.io/): `gem install bundler`. We use bundler to install the right version of [CocoaPods](https://cocoapods.org/) locally.
-3. Install Bundler and CocoaPods dependencies: `bundle install && bundle exec pod install` or `yarn setup-ios-hermes`. In order to use JSC instead of Hermes engine, run: `USE_HERMES=0 bundle exec pod install` or `yarn setup-ios-jsc` instead.
-4. Open the generated `RNTesterPods.xcworkspace`. This is not checked in, as it is generated by CocoaPods. Do not open `RNTesterPods.xcodeproj` directly.
+1. Install [Homebrew](https://brew.sh), which we'll need so we can avoid overwriting important Ruby files used by the operating system
+2. `brew install ruby`, then follow the instructions on screen for adding this new Ruby to your path.
```

Comment:
MacOS comes with a default Ruby version and we support that. Also, to work with iOS, you need a macOS. As of above, we don't really want to enforce more requirements than needed.